pwndbg/arguments.py

```diff
@@ -221,9 +222,18 @@ def format_args(instruction):
 
         # Enhance args display
         if arg.name == "fd" and isinstance(value, int):
-            path = pwndbg.gdblib.file.readlink("/proc/%d/fd/%d" % (pwndbg.gdblib.proc.pid, value))
-            if path:
-                pretty += " (%s)" % path
+            if pwndbg.gdblib.proc.pid is not None:
+                path = pwndbg.gdblib.file.readlink(
+                    "/proc/%d/fd/%d" % (pwndbg.gdblib.proc.pid, value)
```

Comment:
Not a big deal, but maybe we should assign the pid to a variable here since its a getter iirc